auto_install.php

```diff
@@ -0,0 +1,164 @@
+<?php
```

Comment:
We now have a CLI console where classes have replaced physical scripts, please check the `src/Core/Console` folder for existing examples of console commands.